### ISSUE TITLE
Bring back room-local avatar changes in bubble layout

### ISSUE THREAD

**Issue by tastytea** [Nheko-Reborn/nheko#1000]
### The Problem

Without bubbles, I can change my room-specific nick and avatar by clicking on my avatar in the timeline. In the bubble layout, my avatar is not shown.

### The Solution

- The simplest would be to add my avatar to my bubbles, but I gathered that goes against the spirit of the layout?
- The room specific profile could be made accessible from room settings — I don't know, it is kinda related, but also kinda unrelated to that.
- The room specific profile could be made accessible from the meatball menu — Hmm, maybe?

### Alternatives

- Local nick changes are possible with `/roomnick`.
- I can get to the dialog by clicking my avatar in the participants list, but that is unintuitive and tedious in big rooms.
- Switching to un-bubbled layout, change nick / avatar, change back to bubbled layout. Tedious as well.

### Happens in the latest version

- [X] Yes, this feature is still missing.

**Comment by maltee1** [comment]
yeah, bringing the avatar back would indeed go against the spirit of the layout.

I suppose the best way is to add it to the meatball menu.

**Comment by maltee1** [comment]
I'm thinking, wouldn't putting oneself on top of the memberlist solve the problem more or less? It's not as direct, but it would not require to search for oneself.

**Comment by tastytea** [comment]
Yeah, that looks like a good compromise. It's not as intuitive as the timeline-avatar, but not less intuitive than an extra meatball-slice, I think.
